build.gradle.kts

```diff
@@ -172,9 +173,9 @@ tasks {
             copy {
                 from(tarTree(helmDir.file("helm.tar.gz")))
                 into(helmDir)
-                fileMode = 0b111101101
+                filePermissions { unix("rwxr-xr-x") }
             }
-            exec {
+            project.exec {
```

Comment:
'exec(Action<ExecSpec>): ExecResult' is deprecated. This method will be removed in Gradle 9.0. Use ExecOperations.exec(Action) or ProviderFactory.exec(Action) instead.